FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -125,7 +156,16 @@ class IssuesViewController : UITableViewController {
         loadIssues()
     }
 
+    fileprivate func setTitleLabel(location: UserLocation?) {
+        locationButtonItem?.title = location?.locationDisplay ?? "Set Location"
```

Comment:
Need localization for "Set Location" if it doesn't exist already.  (Check `R.strings.localized.`...)